src/main/java/appeng/decorative/solid/BlockChargedQuartzOre.java

```diff
@@ -49,9 +50,10 @@ public BlockChargedQuartzOre()
 	@Override
 	public Item getItemDropped( final IBlockState state, final Random rand, final int fortune )
 	{
-		for( final Item charged : AEApi.instance().definitions().materials().certusQuartzCrystalCharged().maybeItem().asSet() )
+		Optional<Item> item = AEApi.instance().definitions().materials().certusQuartzCrystalCharged().maybeItem();
+		if( item.isPresent() )
 		{
-			return charged;
+			return item.get();
```

Comment:
Ah yes, I forgot about that one. It's used in several places. I'll go back and change it.